topics/microbiome/tutorials/visualisation-ampvis/tutorial.md

```diff
@@ -0,0 +1,623 @@
+---
+layout: tutorial_hands_on
+
+
+title: Visualization workflows for metagenomic amplicon data using the Galaxy framework
+level: Intermediate
+zenodo_link: https://zenodo.org/records/11281381
+questions:
```

Comment:
some ideas: 
* how can we adapt the plots to our research data
* how can we filter the data to show only significant information
* how can we compare multiple visualization methods
* how can we use numerical and categorical metadata for amplicon visualization

Reply:
took all your suggestions